### PR TITLE
DeveloperGuide.adoc: update manual testing

### LINKED ISSUE #98: Update the manual instruction for Developer Guide.
Update the manual instruction for Developer Guide.

### PR DESCRIPTION
**Pull request by Amoscheong97**

### CODE REVIEW

**Review by Exeexe93 — APPROVED** [review]
Look good to merge.

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -799,40 +799,169 @@ Given below are instructions to test the app manually.
...
+.. Test case: Invalid Syntax +
+   Expected: No order is deleted. An error message will be displayed in the response box
+.. Test case: `delete *ANY INDEX NOT FOUND IN LIST*` +
+   Expected: No order is deleted. An error message will be displayed in the response box, indicating that the index cannot be found in the list
+
+=== Edit orders
+
+. You can edit the details of the delivery order by specifying the transaction id, the field you want to change and the value that is required to update
+
+.. Edit Command Syntax: `edit` `*TRANSACTION ID*` `*FIELD*` `*NEW VALUE*`
```
Comment (khsc96) [inline docs/DeveloperGuide.adoc]: I think we should standardise the name of the field either with _ or without.

**Review by khsc96 — CHANGES_REQUESTED** [review]
Minor changes, standardisation issue other than that goodjob!

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -868,41 +872,43 @@ These instructions only provide a starting point for testers to work on; testers
...
 .. Test case: `list` +
    Expected: List all the delivery orders, showing all completed and uncompleted orders.
-.. Test case: `list done` +
+.. Test case: `list` `done` +
    Expected: List all completed delivery orders.
-.. Test case: `list undone` +
+.. Test case: `list` `undone` +
    Expected: List all uncompleted delivery orders.
-.. Test case: `list *ANY WORD OTHER THAN UNDONE AND DONE*` +
+.. Test case: `list` `ANY WORD OTHER THAN UNDONE AND DONE` +
```
Comment (khsc96) [inline docs/DeveloperGuide.adoc]: separate white space by underscore thanks!

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -825,41 +825,45 @@ These instructions only provide a starting point for testers to work on; testers
...
-.. Test case: `delete 1` +
+.. Test case: `delete` `1` +
    Expected: The first order item in the current list will be removed. Details of the deleted order will be displayed in the response box
-.. Test case: `delete 2` +
+.. Test case: `delete` `2` +
    Expected: The second item in the current list will be removed. Details of the deleted order will be displayed in the response box
 .. Test case: Invalid Syntax +
    Expected: No order is deleted. An error message will be displayed in the response box
-.. Test case: `delete *ANY INDEX NOT FOUND IN LIST*` +
+.. Test case: `delete` `ANY INDEX NOT FOUND IN LIST` +
```
Comment (khsc96) [inline docs/DeveloperGuide.adoc]: separate whitespace by underscore! or can simply put INVALID_INDEX

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -825,41 +825,45 @@ These instructions only provide a starting point for testers to work on; testers
...
-.. Test case: `edit 0123456789 name Xuan En` +
-   Expected: Name of the customer of the order with transaction ID 0123456789 will be changed to Xuan En
-.. Test case: `edit 0123456789 phone 99521654` +
-   Expected: Phone number of the customer of the order will be changed to 9952 1654
-.. Test case: `edit 1654789206 address Blk 123 Pasir Ris street 51 #12-23 S510123` +
-   Expected: The address of the customer of the order will be changed to Blk 123 Pasir Ris Street 51 #12-23 S510123
-.. Test case: `edit 9876543210 date 09/08/2020` +
-   Expected: The timestamp of the customer of the order will be rescheduled to 09/08/2020
-.. Test case: `edit 9874639183 date 02/02/2020` (Assuming this date has passed) +
+.. Edit command format: `edit` `INDEX` `PREFIX/NEW VALUE`
```
Comment (khsc96) [inline docs/DeveloperGuide.adoc]: NEW_VALUE, thank you!

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -933,24 +939,28 @@ These instructions only provide a starting point for testers to work on; testers
...
+   
 === Undo previous action
 
 . Undo the user’s previous action and revert back to the previous state. Usually in the case where the user has accidentally deleted an order
 
 .. Prerequisites : Execute any valid command that edits the list before calling the `undo` command
 .. Test case: `undo` +
    Expected: The original list will be replaced with the current new list. The previous state will be restored
-.. Test case: `undo <Any other words as the second argument>` +
+.. Test case: `undo` `ANY OTHER WORD` +
```
Comment (khsc96) [inline docs/DeveloperGuide.adoc]: Perhaps can use INVALID_SYNTAX instead

**Review by khsc96 — CHANGES_REQUESTED** [review]
Please have a look at the comments i've given and change some whitespace to underscore accordingly thankyou!

**Review by khsc96 — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**